mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java

```diff
@@ -288,6 +288,14 @@ public void onClick(View v) {
         });
     }
 
+    public boolean getToggleStatus() {
+        if (PrefManager.getUserStatus() == Constants.USER_OFFLINE) {
+            return true;
+        } else {
+            return false;
+        }
+    }
```

Comment:
You can reduce the number of line codes by simply doing this:
`  public boolean getToggleStatus() {
return PrefManager.getUserStatus() == Constants.USER_OFFLINE 
}`

Reply:
Yeah will do that